quinn-proto/src/connection/mod.rs

```diff
@@ -2227,15 +2228,8 @@ impl Connection {
             self.stats.frame_rx.record(&frame);
 
             let _guard = span.as_ref().map(|x| x.enter());
-            // Check for ack-eliciting frames
```

Comment:
This group of changes could technically be a commit on its own, right?

Reply:
Isn't it?